libpod/container_api.go

```diff
@@ -203,7 +203,7 @@ func (c *Container) Kill(signal uint) error {
 	}
 
 	if c.state.State != ContainerStateRunning {
-		return errors.Wrapf(ErrCtrStateInvalid, "can only kill running containers")
+		return errors.Wrapf(ErrCtrStateInvalid, "can only kill running containers. %s in state %s", c.ID(), c.state.State.String())
```

Comment:
suggest s/in state/is in state/

Reply:
Currently, it's consistent with another error. I can change both if you like?

Reply:
It's a "six of one, half dozen of another" deal.  I prefer "is in", but both are correct.  If you decide to change one, then please do change the other.

Reply:
Done :heavy_check_mark: